rotkehlchen/chain/ethereum/oracles/uniswap.py

```diff
@@ -252,6 +269,21 @@ def get_price(
         price = FVal(reduce(mul, [item.price for item in prices_and_tokens], 1))
         return Price(price)
 
+    def _resolve_assets(self, assets: list[Asset | AssetWithOracles]) -> list[AssetWithOracles]:
```

Comment:
can you make this a tuple `(AssetWithOracles | Asset, AssetWithOracles | Asset)`